unittests/backends/QPPDMTester.cpp

```diff
@@ -128,4 +128,200 @@ CUDAQ_TEST(QPPTester, checkDensityOrderingBug) {
     EXPECT_EQ(1, qppBackend.mz(q2));
     EXPECT_EQ(0, qppBackend.mz(q3));
   }
-}
\ No newline at end of file
+}
+
+// CUDAQ_TEST(QPPTester, checkSetStateDensity) {
```

Comment:
Will this be enabled with another PR?

Reply:
This refers to the test below now. vv

Reply:
Yes, this has been moved to a branch where it will sit until we support state initialization on the density simulator